test/parallel/test-repl-reset-event.js

```diff
@@ -61,7 +64,8 @@ function testResetGlobal() {
     assert.strictEqual(
       context.foo,
       42,
-      '"foo" property is missing from REPL using global as context'
+      '"foo" property is different from REPL using global as context.' +
```

Comment:
Nit: Probably should be a space at the end of the string on this line or else a space at the beginning of the string on the next line? Same for two other instances above.

Reply:
Good point, tell me what'd prefer :P

Reply:
@AdriVanHoudt I think we prefer @Trott's suggested style for most of the codebase :)

Reply:
@AdriVanHoudt Space at the end is more typical throughout the code base as far as I've noticed, so go with that I guess? But either would be fine.